index.js

```diff
@@ -0,0 +1,22 @@
+document.addEventListener("DOMContentLoaded", () =>{
+    // let deckId =""
+    const fetchData = async () => {
+        let res = await axios.get("https://deckofcardsapi.com/api/deck/new/")
+        let deck_id = res.data.deck_id
+
+        let shuffled = axios.get(`https://deckofcardsapi.com/api/deck/${deck_id}/shuffle/`)
```

Comment:
You can use this url to get a shuffled new deck, then here you can just save the deckId `https://deckofcardsapi.com/api/deck/new/shuffle/?deck_count=1`